src/quo2/components/avatars/icon_avatar.cljs

```diff
@@ -16,9 +17,10 @@
         circle-color   (colors/custom-color color 50 opacity)
         icon-color     (colors/custom-color-by-theme color 50 60)
         icon-size      (case size
-                         :big    20
-                         :medium 16
-                         :small  12)]
+                         :size/s-48 20
+                         :size/s-32 20
+                         :size/s-24 16
+                         :size/s-20 12)]
```

Comment:
It's worth always adding add a default value for `case` in status-mobile, otherwise the user may get a blank screen. This can easily happen if the DS evolves to have a new size and we forget to update this code.

Reply:
fixed, thanks